web-gateway/app/views/createUser.scala.html

```diff
@@ -9,9 +9,10 @@
     @foundationForm(createUser, routes.Main.createUser()) {
         <div class="row">
             <div class="medium-6 columns">
+              <p>Enter username in the box below and click on 'Create User' to create a new user profile.</p>
```

Comment:
Could you extract this message into the `web-gateway/conf/messages` file? Perhaps call it `createUserInstructions`. Also perhaps not necessary to instruct users to click create user, but rather just state what this form is for and what it will do, maybe "Here you can create a new user who can participate in and create auctions."